.gitignore

```diff
@@ -1,5 +1,5 @@
 /config/dynamic.php
-/config/app.php
+/config/app.php # TODO: this should not be here??
```

Comment:
This file is auto-generated and should be out of the repo.

Reply:
should it be removed from here then?
https://github.com/cypht-org/cypht/blob/master/config/app.php

Reply:
Ah, you are right - app is not build dynamically, it is the config/dynamic.php one that should be gitignored. Please remove config/app.php from .gitignore file - it is a valid repo file.